### PR TITLE
Replace old plan icons with the new ones

### PR DESCRIPTION
**Pull request by adamziel**
#### Changes proposed in this Pull Request

* Introduces a new set of plan icons that we want to start using as a result of eCommerce and Blogger plan merge (p9jf6J-14t-p2)

Important note - this PR just replaces the icons, so the change is going to be live for everyone immediately after merging this. There's no distinction between the test group and other users.

#### Testing instructions

* Go to /plan and plans page in the signup flow, and confirm you can see the following icons and colors for corresponding tabs:

zrzut ekranu 2018-11-23 o 14 21 57
zrzut ekranu 2018-11-23 o 14 22 01

* Do the same for Jetpack, confirm we still use old icons and colors there:

zrzut ekranu 2018-11-21 o 14 48 39

### CODE REVIEW

File: client/my-sites/plans-features-main/style.scss [hunk 1]
```diff
@@ -22,3 +22,17 @@
 		color: $blue-wordpress;
 	}
 }
+
+.plans-features-main__group.is-customer-personal {
```
Comment (jancavan) [inline client/my-sites/plans-features-main/style.scss]: This might be a little too nested.

**Comment by obenland** [comment]
Do you want to add updating the banner color to this PR?
https://wpcalypso.wordpress.com/devdocs/design/banner

It's what I've been holding off from committing #28362 for

**Comment by jancavan** [comment]
@adamziel In this changeset: https://github.com/Automattic/wp-calypso/pull/28715/commits/4dd56bd560b248527135afd43e96c8ab9fb60f06:

Do some of them need to be minified?

What are the trial icons being used for?

**Comment by adamziel** [comment]
@jancavan god spot - it looks like these trial icons are unused so instead of moving them to a subdirectory I went ahead and removed them. Also minification is now taken care of.

**Comment by adamziel** [comment]
@obenland I will update these tomorrow, thanks!

**Comment by jancavan** [comment]
@adamziel I think the border colors need to be updated as well. Doing that now and will push changes in a bit. I can also take a look at banner component icons/colors update.

**Comment by jancavan** [comment]
Updated the border colors of each column to match their icon colors in: https://github.com/Automattic/wp-calypso/pull/28715/commits/621230e27b8d2c188b34171343c18edfeae8bd71

screenshot 2018-11-21 09 53 50
screenshot 2018-11-21 09 53 53

**Comment by jancavan** [comment]
I think we're able to just use our pre-defined Calypso colors instead of custom hex values. Updated in: https://github.com/Automattic/wp-calypso/pull/28715/commits/bc613f186e774e81a042aadc5028c7f6e5edb05e

**Comment by jancavan** [comment]
As mentioned in this PR: https://github.com/Automattic/wp-calypso/pull/28362, banner component colors have been updated as well.

**Before:**
screenshot 2018-11-21 10 10 53

**After:**
screenshot 2018-11-21 10 15 17

**Comment by adamziel** [comment]
Nice @jancavan! It looks good to me, anything else we need before merging this PR? Review appreciated!

**Comment by jancavan** [comment]
@adamziel I don't see the ecommerce banner upgrade here though: https://wpcalypso.wordpress.com/devdocs/design/banner

Not sure if the banner already exists, but just hasn't been added to Devdocs? I don't think it needs to, but just wanted to make sure.

Otherwise, visually, LGTM 👍

**Comment by jancavan** [comment]
I also wanted to note that some of the icons are duplicates - for example: `plan-personal.svg` vs `plan-personal-circle.svg`

I think we can just use one or the other? Can just be addressed in a separate PR though.

**Comment by adamziel** [comment]
Just updated this PR to make sure Jetpack colors will remain the same since we're not updating Jetpack icons just yet. Should be good to go!

**Pull request merged**